From e5bb23969afa2bb3cab95ef1ede0f904d602423e Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sat, 27 Jan 2018 22:30:15 +0100
Subject: [PATCH 1/4] CarrierConfig: Add selected configs for national roaming

This is a squash of following changes:

Author: dh-harald <github@criminal.hu>
Date:   Fri Nov 21 10:00:36 2014 +0000

    Vectone UK (MNC01) considered non-roaming on T-Mobile (MNC30)

    Change-Id: Id74c854f958be721be12451a124cfd2f8b150255

Author: neatchee <neatchee@gmail.com>
Date:   Thu Aug 21 17:25:13 2014 -0700

    Lycamobile (MNC04) considered non-roaming on Vodafone (MNC01)

    Change-Id: Icff916672e5ed666adf2cdfeb26d24bf43b64683

Author: neatchee <neatchee@gmail.com>
Date:   Thu Aug 21 17:22:03 2014 -0700

    Vectone/Delight (MNC07) considered non-roaming on NOS (MNC03)

    Change-Id: I743935b94bb531d78b784550e3028378cdc01959

Author: neatchee <neatchee@gmail.com>
Date:   Fri Jun 27 14:52:15 2014 -0700

    Noverca (MNC07) considered non-romaing on TIM (MNC01)

    Change-Id: I473e1f88db0fe0d5e1799f6d6d892649abb7dc74

Author: neatchee <neatchee@gmail.com>
Date:   Thu May 15 16:17:42 2014 -0700

    Tele2 considered non-roaming on T-Mo

    Change-Id: Ibb05f277b701861b8716168b54e96f63127e686d

Author: neatchee <neatchee@gmail.com>
Date:   Wed May 21 16:56:22 2014 -0700

    Saunalahti considered non-roaming on Elisa

    Change-Id: I51b554cf578c6027d52ad6e111e41d269c1f579f

Author: Phil Eichinger <phil@zankapfel.net>
Date:   Tue May 13 10:00:47 2014 +0200

    National Roaming: Allow T-Mobile AT (MNC03) to roam on 3 AT (MNC10) in Austria

    Change-Id: Ief827259fdcb5bfa13bd643f80a8547a56cc934c

Author: Arne Coucheron <arco68@gmail.com>
Date:   Tue May 13 00:36:20 2014 +0200

    Allow 3 AT (mnc05) to roam on T-Mobile in Austria

    Change-Id: Ie1a62db9a48cfaf594ddda7b413299a9b587ddd4

Author: d95andek <andreas@gunneras.se>
Date:   Fri Jan 17 22:20:33 2014 +0100

    core: added national roaming info for Spring Mobil in Sweden...

    Spring Mobil users in Sweden (MCC 240, MNC 10)
    can roam on Sweden 3G (MNC 05) and on Sweden 2G (MNC 24).

    Change-Id: Ie27ad3561de32557a1713d4962fcc9ec52a8b7fe

Author: broodplank <guichardflorent@gmail.com>
Date:   Thu Dec 19 00:15:21 2013 +0100

    core: added national roaming info for Tesco Mobile in Ireland.

    Users of Tesco Mobile in Ireland(MCC 272 MNC 11)
      can roam on O2 Ireland (MNC 02)

    Change-Id: Ie0b32d51511c2bbc726c43cdec78332397d1ecb9

Author: Nebojsa Cvetkovic <nebkat@gmail.com>
Date:   Sun Dec 15 19:23:49 2013 +0000

    National roaming info for 3 in Ireland

    Change-Id: I451831f4144b72feac1a38ef1e8693cb947d6d7e

Author: maxwen <max.weninger@gmail.com>
Date:   Sat Dec 7 17:33:32 2013 +0100

    base: added national roaming info for Play in Poland

    Cherry-picked from OmniROM:
    https://gerrit.omnirom.org/#/c/3382/

    Users of Play in Poland (MCC 260 MNC 06) can roam on
    Plus, T-Mobile, Orange (MNCs 01/02/03)

    Change-Id: I8ea62b0ca40bac9ced11802ee8ef583d6392a199

Change-Id: I4790d3a167e93874d8da3062fe9529ab0c044bbc
---
 assets/carrier_config_20402.xml |  8 ++++++++
 assets/carrier_config_22207.xml |  8 ++++++++
 assets/carrier_config_23203.xml |  3 +++
 assets/carrier_config_23205.xml |  8 ++++++++
 assets/carrier_config_23401.xml |  8 ++++++++
 assets/carrier_config_24010.xml |  9 +++++++++
 assets/carrier_config_24421.xml |  8 ++++++++
 assets/carrier_config_26006.xml | 10 ++++++++++
 assets/carrier_config_26804.xml |  8 ++++++++
 assets/carrier_config_26807.xml |  8 ++++++++
 assets/carrier_config_27205.xml |  8 ++++++++
 assets/carrier_config_27211.xml |  8 ++++++++
 12 files changed, 94 insertions(+)
 create mode 100644 assets/carrier_config_20402.xml
 create mode 100644 assets/carrier_config_22207.xml
 create mode 100644 assets/carrier_config_23205.xml
 create mode 100644 assets/carrier_config_23401.xml
 create mode 100644 assets/carrier_config_24010.xml
 create mode 100644 assets/carrier_config_24421.xml
 create mode 100644 assets/carrier_config_26006.xml
 create mode 100644 assets/carrier_config_26804.xml
 create mode 100644 assets/carrier_config_26807.xml
 create mode 100644 assets/carrier_config_27205.xml
 create mode 100644 assets/carrier_config_27211.xml

diff --git a/assets/carrier_config_20402.xml b/assets/carrier_config_20402.xml
new file mode 100644
index 0000000..f885476
--- /dev/null
+++ b/assets/carrier_config_20402.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="20416" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_22207.xml b/assets/carrier_config_22207.xml
new file mode 100644
index 0000000..b680278
--- /dev/null
+++ b/assets/carrier_config_22207.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="22201" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_23203.xml b/assets/carrier_config_23203.xml
index e8cacc5..ddc94fc 100644
--- a/assets/carrier_config_23203.xml
+++ b/assets/carrier_config_23203.xml
@@ -2,5 +2,8 @@
 <carrier_config_list>
     <carrier_config>
         <boolean name="display_hd_audio_property_bool" value="false" />
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="23210" />
+        </string-array>
     </carrier_config>
 </carrier_config_list>
diff --git a/assets/carrier_config_23205.xml b/assets/carrier_config_23205.xml
new file mode 100644
index 0000000..8707a71
--- /dev/null
+++ b/assets/carrier_config_23205.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="23203" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_23401.xml b/assets/carrier_config_23401.xml
new file mode 100644
index 0000000..c3f8978
--- /dev/null
+++ b/assets/carrier_config_23401.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="23430" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_24010.xml b/assets/carrier_config_24010.xml
new file mode 100644
index 0000000..c888f53
--- /dev/null
+++ b/assets/carrier_config_24010.xml
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="2">
+            <item value="24005" />
+            <item value="24024" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_24421.xml b/assets/carrier_config_24421.xml
new file mode 100644
index 0000000..c4f9fb7
--- /dev/null
+++ b/assets/carrier_config_24421.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="24405" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_26006.xml b/assets/carrier_config_26006.xml
new file mode 100644
index 0000000..4d55534
--- /dev/null
+++ b/assets/carrier_config_26006.xml
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="3">
+            <item value="26001" />
+            <item value="26002" />
+            <item value="26003" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_26804.xml b/assets/carrier_config_26804.xml
new file mode 100644
index 0000000..e125707
--- /dev/null
+++ b/assets/carrier_config_26804.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="26801" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_26807.xml b/assets/carrier_config_26807.xml
new file mode 100644
index 0000000..1d62197
--- /dev/null
+++ b/assets/carrier_config_26807.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="26803" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_27205.xml b/assets/carrier_config_27205.xml
new file mode 100644
index 0000000..b69b442
--- /dev/null
+++ b/assets/carrier_config_27205.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="27201" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
diff --git a/assets/carrier_config_27211.xml b/assets/carrier_config_27211.xml
new file mode 100644
index 0000000..876967d
--- /dev/null
+++ b/assets/carrier_config_27211.xml
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
+<carrier_config_list>
+    <carrier_config>
+        <string-array name="non_roaming_operator_string_array" num="1">
+            <item value="27202" />
+        </string-array>
+    </carrier_config>
+</carrier_config_list>
-- 
2.17.1

